infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -348,9 +348,21 @@ func getKVFromOutputString(v string) (string, string, error) {
 // GetStringOutput returns TF output for a given key as string.
 // It fails test if given key does not output a primitive.
 func (b *TFBlueprintTest) GetStringOutput(name string) string {
+	// allow only parallel reads as Terraform plugin cache isn't concurrent safe
+	rUnlockFn := b.rLockFn()
+	defer rUnlockFn()
 	return terraform.Output(b.t, b.GetTFOptions(), name)
 }
 
+// GetStringOutputList returns TF output for a given key as list.
+// It fails test if given key does not output a primitive.
+func (b *TFBlueprintTest) GetStringOutput(name string) string {
```

Comment:
```suggestion
func (b *TFBlueprintTest) GetStringOutputList(name string) []string {
```

Reply:
Thanks - Was just about to fix the return type :)